### ISSUE TITLE
Cannot test file accessibility

### ISSUE THREAD

**Issue by ramiromagno** [r-lib/testthat#763]
I am trying to test a function named `read_transcript_expression` that internally tests for input accessibility with:

```R
 # check if file is readable
  if(file.access(filename, mode = 0))
    stop(filename, " is not readable. Please check file permissions.")
```

My approach is to have this test group in one of my test files:

```R
test_that("read_transcript_expression exits with an error when input file is not readable", {
  expect_error(read_transcript_expression("not-readable.txt"), "not-readable.txt is not readable. Please check file permissions.")
})
```
File `"not-readable.txt"` lives in `tests/testthat` alongside with the test containing the aforementioned test group.
However, it seems that after running `devtools::use_testthat()` testthat misses the file `"not-readable.txt"`, never getting to the point of actually testing its permissions:

```
test-read-transcript-expression.R:8: failure: read_transcript_expression exits with an error when input file is not readable
`read_transcript_expression("not-readable.txt")` threw an error with unexpected message.
Expected match: "not-readable.txt is not readable. Please check file permissions."
Actual message: "file not found: not-readable.txt"
```
